docs/src/variables.md

```diff
@@ -470,7 +470,21 @@ Dict{Symbol,Symmetric{JuMP.VariableRef,Array{JuMP.VariableRef,2}}} with 2 entrie
 
 ## Deleting variables
 
-**TODO(@odow):** explain how to delete variables.
+Finally, JuMP supports the deletion of optimization variables. To delete
+variables, we can use the `JuMP.delete` method. We can also check whether `x`
```

Comment:
What happens when the variable that's deleted already appears in constraints?

Reply:
Urgh. Solver-specific? At least in direct mode. What does the caching optimizer do? @blegat?